server/routes/dashboardCollectionOverview.tsx

```diff
@@ -43,9 +47,10 @@ app.get('/dash/collection/:collectionSlug/overview', async (req, res, next) => {
 		}
 
 		const overviewData = await getCollectionOverview(initialData);
-		const hasSubmissionWorkflow = await collectionHasSubmissionWorkflow(
+		const submissionWorkFlow = await collectionHasSubmissionWorkflow(
 			overviewData.collection.id,
 		);
+		const hasSubmissionWorkflow = submissionWorkFlow && submissionWorkFlow.enabled;
```

Comment:
We should prob rename this `hasEnabledSubmissionWorkflow` just to be 100% clear.